src/src/Commands/InternalCommands.cpp

```diff
@@ -416,6 +417,7 @@ bool executeInternalCommand(command_case_data & data)
         COMMAND_CASE_A(      "servo", Command_Servo,          3); // Servo.h
       }
       COMMAND_CASE_A("status", Command_GPIO_Status,          2); // GPIO.h
+      COMMAND_CASE_A("serialwritehex", Command_Serial_WriteHex,  -1); // Serial.h
```

Comment:
You should not have a plugin command and a global command using the same command name.
To be honest, this is a rather specific command, so I don't think it is perfectly fine to have it part of Ser2Net.
Or is there a good reason not to use Ser2Net when you need this?

Reply:
And looking at the alphabetic order, it should probably have gone above current line 416